sources/Form/Field/FileUploadField.php

```diff
@@ -134,6 +142,25 @@ public function SetDownloadEndpoint(string $sDownloadEndpoint)
 		return $this;
 	}
 
+	/**
+	 * @return string|null
+	 */
+	public function GetDisplayEndpoint(): ?string
+	{
+		return $this->sDisplayEndpoint;
+	}
+
+	/**
+	 * @param string $sDisplayEndpoint
+	 *
+	 * @return $this
+	 */
+	public function SetDisplayEndpoint(string $sDisplayEndpoint): FileUploadField
```

Comment:
```suggestion
	public function SetDisplayEndpoint(string $sDisplayEndpoint): static
```

I'll rather return `static` than the hardcoded class, otherwise it will crash if the class is extended.
_Note: Using static is only possible since PHP 8.0, that is way we didn't put class type hint in previous iTop versions and only put `@return $this` in the PHPDoc_